apps/extension/src/App/Accounts/ViewingKey.tsx

```diff
@@ -14,8 +14,9 @@ export const ViewingKey = (): JSX.Element => {
       <Stack full gap={GapPatterns.TitleContent}>
         <PageHeader title="Viewing Key" />
         <p className="text-white">
-          Your viewing key grants the holder access to all your balances and
-          transaction data. Please keep it secure to protect your data.
+          Your viewing key grants the holder the ability to see to all your
```

Comment:
can you replace `to see to all` by `to see all` (removing the second `to`) ?

ref: https://github.com/anoma/namada-interface/issues/1714#issuecomment-2677952091